configure.ac

```diff
@@ -167,9 +167,9 @@ AS_IF([test x"$strip" == x"yyy"], [
   AC_MSG_NOTICE([Not using compiler options to reduce binary size!])
 )
 
-AC_CHECK_PROG([BASH],[bash],[yes])
+AC_CHECK_PROG([BASH_SHELL],[bash],[yes])
 AS_IF(
-    [test "x${BASH}" == x"yes"],
+    [test "x${BASH_SHELL}" == x"yes"],
```

Comment:
@williamcroberts yeah, the check was working on the Travis CI because Ubuntu uses dash as /bin/sh and so the BASH variable wasn't set.

So ironically, the check for bash only worked when the configure script wasn't executed from bash :smiley: